### PR TITLE
36 make input field work mobile

### LINKED ISSUE #36: Make input field work mobile

### PR DESCRIPTION
**Pull request by Gormiee**
added inputfield that works with logical name, navbar for mobile

### CODE REVIEW

File: src/UICrafter.Core/Components/BaseComponents/BaseInput.razor [hunk 1]
```diff
@@ -1,8 +1,17 @@
-<MudTextField T="string" Class="mb-2" Variant="Variant.Outlined" Label="@Label" @bind-Value="@Value" @onfocus="OnFocus" />
+<MudTextField
+    T="string"
+    Class="mb-2"
+    Variant="Variant.Outlined"
+    Label="@Label"
+    @bind-Value="@Value"
```
Comment (Nephim) [inline src/UICrafter.Core/Components/BaseComponents/BaseInput.razor:6]: You should use the Value and ValueChanged not TextChanged

File: src/UICrafter.Core/Components/BaseComponents/BaseLoadMetaDataList.razor [hunk 2]
```diff
@@ -37,6 +37,27 @@ else
     protected virtual void HandleClick(AppView view) { }
     protected virtual string? SetupHref(AppView view) { return null; }
 
+    // protected override async void OnParametersSet()
```
Comment (Aagaard00) [inline src/UICrafter.Core/Components/BaseComponents/BaseLoadMetaDataList.razor]: AD!!!

File: src/UICrafter.Mobile/Components/Layout/MainLayout.razor [hunk 3]
```diff
@@ -32,8 +32,39 @@
...
+        <Authorized>
+            <footer class="footer">
+                <NativeNavbar />
+            </footer>
+        </Authorized>
+    </AuthorizeView>
 </div>
 
+<style>
+    .page {
```
Comment (Aagaard00) [inline src/UICrafter.Mobile/Components/Layout/MainLayout.razor]: Move style and no comments plz

File: src/UICrafter.Mobile/Components/Layout/NativeNavbar.razor [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<MudToolBar Class="justify-space-between">
+    <MudIconButton Icon="@Icons.Material.Outlined.Menu" Color="@GetIconColor("Menu")" Class="@GetIconClass("Menu")" Href="/" OnClick="@(() => SelectIcon("Menu"))" />
+    <MudIconButton Icon="@Icons.Material.Outlined.Add" Color="@GetIconColor("Add")" Class="@GetIconClass("Add")" Href="appview" OnClick="@(() => SelectIcon("Add"))" />
+    <MudIconButton Icon="@Icons.Material.Outlined.Edit" Color="@GetIconColor("Edit")" Class="@GetIconClass("Edit")" Href="authpage" OnClick="@(() => SelectIcon("Edit"))" />
+</MudToolBar>
+
+@code {
+    private string? selectedIcon;
```
Comment (Aagaard00) [inline src/UICrafter.Mobile/Components/Layout/NativeNavbar.razor:8]: See if there is an existing Focus thingy in MudBlazor

**Review by Aagaard00 — APPROVED** [review]
LGTM, maybe there is an existing way to do the focus in the app bar thingy

**Review by Nephim — APPROVED** [review]
A minor thing, there is a lot of stuff I would like to change down the line but if we dont get around to it its okay.

**Pull request merged**